docs/src/guide/measure.md

```diff
@@ -274,21 +274,43 @@ Each method is limited on the dimension of parameter and/or the type of set
 that it can apply for. For the details of what each method type means, refer to the corresponding
 docstrings.
 
-| Evaluation Method              | Uni/Multi-Variate? | Set Type                              |
-|:------------------------------:|:------------------:|:-------------------------------------:|
-| [`Automatic()`](@ref)            | Both               | Any                                 |
-| [`UniTrapezoid()`](@ref)         | Both               | [`IntervalSet`](@ref)               |
-| [`UniMCSampling()`](@ref)        | Univariate         | Finite [`IntervalSet`](@ref)        |
-| [`UniIndepMCSampling()`](@ref)   | Univariate         | Finite [`IntervalSet`](@ref)        |
-| [`Quadrature()`](@ref)           | Univariate         | [`IntervalSet`](@ref)               |
-| [`GaussLegendre()`](@ref)        | Univariate         | Finite [`IntervalSet`](@ref)        |
-| [`GaussRadau()`](@ref)           | Univariate         | Finite [`IntervalSet`](@ref)        |
-| [`GaussJacobi()`](@ref)          | Univariate         | Finite [`IntervalSet`](@ref)        |
-| [`GaussLobatto()`](@ref)         | Univariate         | Finite [`IntervalSet`](@ref)        |
-| [`GaussLaguerre()`](@ref)        | Univariate         | Semi-infinite [`IntervalSet`](@ref) |
-| [`GaussHermite()`](@ref)         | Univariate         | Infinite [`IntervalSet`](@ref)      |
-| [`MultiMCSampling()`](@ref)      | Multivariate       | Finite [`IntervalSet`](@ref)        |
-| [`MultiIndepMCSampling()`](@ref) | Multivariate       | Finite [`IntervalSet`](@ref)        |
+The eval method `FEGaussLobatto` uses integral summation to take in specified supports as points
+when evaluating an integral. All other methods will choose their own supports based on the number of nodes
+provided. This method will take in the user supports, and create generative supports along each interval 
+and match them with corresponding coefficients. An example of what this looks like can be seen below. 
+
+![Image](docs\src\assets\Test.png)
+
+```jldoctest FEGaussLobatto
+julia> mref2 = integral(y^2 + u^2, t, num_nodes = 3, eval_method = FEGaussLobatto())
+∫{t ∈ [0, 10]}[y(t)² + u(t)²]
+```
```

Comment:
This doctest fails since it is not a minimum working example. It also would be good to show how this works with the supports via `expand(mref2)` (I would set it up to match the figure). More info about doctests is available at https://juliadocs.github.io/Documenter.jl/stable/man/doctests/.